reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md

```diff
@@ -586,12 +586,18 @@ sudo snap remove powershell-preview
 
 ```sh
 # Download & Install prerequisites
-sudo apt-get install libunwind8 libicu55
-wget http://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.0.0_1.0.1t-1+deb8u6_amd64.deb
-sudo dpkg -i libssl1.0.0_1.0.1t-1+deb8u6_amd64.deb
+wget http://ftp.us.debian.org/debian/pool/main/i/icu/libicu57_57.1-9_amd64.deb
+dpkg -i libicu57_57.1-9_amd64.deb
+apt-get install -y curl gnupg apt-transport-https
```

Comment:
I had to run `apt-get update` before this command.

Reply:
True, before line 591. I will fix that.